src/components/AddressLookup/index.tsx

```diff
@@ -126,7 +145,7 @@ const AddressLookup: FunctionComponent<AddressLookupProps> = ({
         setActiveOption(0);
         setAddressOptions(data.Items || []);
       })
-      .catch(err => console.error(err));
+      .catch(onAddressError);
```

Comment:
I think you want to execute the error 

`.catch((e: Error) => onAddressError(e));`